my/react-props-app/src/App.js

```diff
@@ -0,0 +1,24 @@
+import logo from './logo.svg';
+import './App.css';
+import './CustomButton.css'
```

Comment:
This would usually be imported in `CustomButton.js` to keep the CSS and JSX together, therefore easier to maintain.